### DISCUSSION TITLE
Autopilot cursor sucks

### DISCUSSION THREAD

**Discussion by LikiaBili** [ppy/osu#27011] [Ideas]
Autopilot sucks rn, i wanted to use autopilot to practice and lower my unstable rate, but i kept getting misses because i clicked when the cursor is not on the circle, it just lowers the speed when it approaching the destination circle and when you play the game, you wont really do that kind of speed reducion, and it just functions like the accuracy requirement is higher, please add a setting to make the cursor position affects less

**Comment by Rekunan** [comment]
For now, I would recommend against using autopilot to train ur tapping, since there's a significant difference in normal play due to tap-aim sync.
